ci-operator/jobs/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master-presubmits.yaml

```diff
@@ -121,3 +121,76 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-gcp,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/e2e-gcp-ovn
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-kni-cnf-features-deploy-master-e2e-gcp-ovn
+    rerun_command: /test e2e-gcp-ovn
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --branch=master
+        - --give-pr-author-access-to-namespace=true
+        - --lease-server=http://boskos
+        - --org=openshift-kni
+        - --repo=cnf-features-deploy
+        - --resolver-address=http://ci-operator-configresolver-ci.svc.ci.openshift.org
+        - --secret-dir=/usr/local/e2e-gcp-ovn-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2e-gcp-ovn
+        - --template=/usr/local/e2e-gcp-ovn
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_NETWORK_TYPE
+          value: OVNKubernetes
```

Comment:
is there a reason prowgen can't do this for us?

the code's in this repo. 
https://github.com/openshift/ci-tools/tree/master/cmd/ci-operator-prowgen

Reply:
Yes I know, but the network type is not configurable on config side as per https://github.com/openshift/ci-tools/blob/master/CONFIGURATION.md

I think it's about keeping a subset of features for not overcomplicating it

Reply:
cc @stevekuznetsov